eng/internal/Directory.Build.targets

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
```

Comment:
What's the value of this file? Can you add a comment?

Reply:
I think he's trying to avoid Directory.Build.Targets from the root to be imported. However, this is not needed to avoid that. You can just include a property in your Directory.Build.props to avoid that: 
https://github.com/dotnet/corefx/blob/c5118d8a571b75b6333893b7be8d4a0c77dedd1e/pkg/Microsoft.Windows.Compatibility/tests/Directory.Build.props#L3